lang/php/lib/avro/data_file.php

```diff
@@ -209,12 +209,12 @@ class AvroDataIOReader
   /**
    * @var string
    */
-  private $sync_marker;
+  public $sync_marker;
```

Comment:
I think we still want to keep this private, as we don't want to expose this outside the object. WDYT?

Reply:
Without this change the code would fail on line 460 😄